snakemake/sourcecache.py

```diff
@@ -266,9 +281,10 @@ def infer_source_file(path_or_uri, basedir: SourceFile = None):
         if not os.path.isabs(path_or_uri) and basedir is not None:
             return basedir.join(path_or_uri)
         return LocalSourceFile(path_or_uri)
+
     if path_or_uri.startswith("git+file:"):
         root_path, file_path, ref = split_git_path(path_or_uri)
-        return LocalGitFile(root_path, file_path, ref=ref)
+        return LocalGitFile("file:/" + root_path, file_path, ref=ref)
```

Comment:
Why adding this prefix here? The class name already says that it is a local thing?

Reply:
refactored and removed

Reply:
refactored and removed